src/App.js

```diff
@@ -11,17 +11,28 @@ import GlobalStyles from './GlobalStyles';
 import theme from './theme';
 
 function App() {
+  const refWorks = useRef(null);
+  const refAbout = useRef(null);
+
+  const scrollToWorks = () => {
+    refWorks.current.scrollIntoView();
+  };
+
+  const scrollToAbout = () => {
+    refAbout.current.scrollIntoView();
+  };
```

Comment:
Great job overall. You don't need this manual scrolling implementation though. It's easier to simply use the Browser's built-in behavior of scrolling to an element with a certain id when that id is in the hash. You can use for example [react-router-hash-link](https://www.npmjs.com/package/react-router-hash-link) to achieve this with react-router